en/codelists-guides/codelist-api.rst

```diff
@@ -107,14 +106,14 @@ Working with GitHub Directly
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 The codelist source is now on GitHub:
 
-Embedded Codelists
+Core (Previously Embedded) Codelists
 ------------------
 As of version 1.04, each version of the standard has it's own branch in this repository. Branches are named version-<version-number> e.g. version-1.05.
 So, for example, the version 1.05 codelists can be found at:
 
 - https://github.com/IATI/IATI-Codelists/tree/version-1.05/xml
 
-Non-Embedded Codelists
+Non-Core and Replicated (Previously Non-Embedded Codelists
```

Comment:
I think that a convention should be decided and stuck to - if it'd be a big chunk of work to normalise _everything everywhere_, at least focus on making them the same for this 'chunk' of work. Non-Core vs NonCore

Reply:
It seems that Non-Core is used the majority of the time

Reply:
This line also contains an open bracket that isn’t closed.